indigo/indigo/src/main/scala/indigo/platform/audio/SoundSwitch.scala

```diff
@@ -0,0 +1,14 @@
+package indigo.platform.audio
+
+enum SoundSwitch derives CanEqual:
+  /** Stop all sounds, not only the previous same sound
+    */
+  case StopAll
+
+  /** Stop only the previous same sound.
+    */
+  case StopPreviousSame
+
+  /** Continue all previous sounds
+    */
+  case Continue
```

Comment:
I agree that `SoundSwitch` as a name is a bit clunky. How about something like either `SoundOverlap` or `PlaybackPolicy`?

Reply:
Renamed in https://github.com/PurpleKingdomGames/indigo/pull/877/commits/980d8e0093f9abddb244551432259bc33775c7da